docs/training/working-with-data-and-model-artifacts.md

```diff
@@ -0,0 +1,87 @@
+(working-with-data-and-model-artifacts)=
+# Working with Data and Model Artifacts
+
+When running a training job, you will need to pass in data to train and save the resulting model. Both the data and model can be considered [Artifacts](https://docs.mlrun.org/en/latest/store/artifacts.html) in MLRun. In the context of an ML pipeline, the data would be an `input` and the model would be an `output`.
+
+Consider the following snippet from a pipeline in the [Build and run automated ML pipelines and CI/CD](https://docs.mlrun.org/en/latest/tutorial/04-pipeline.html#build-and-run-automated-ml-pipelines-and-ci-cd) section of the docs:
```

Comment:
Consider the following snippet from a pipeline in the [Build and run automated ML pipelines and CI/CD](../tutorial/04-pipeline.html#build-and-run-automated-ml-pipelines-and-ci-cd) section of the docs: